integrations/weaviate/src/haystack_integrations/document_stores/weaviate/document_store.py

```diff
@@ -294,6 +292,16 @@ def _to_data_object(self, document: Document) -> Dict[str, Any]:
                 "Document %s has the unsupported `_split_overlap` meta field. It will be ignored.", data["_original_id"]
             )
 
+        if "dataframe" in data:
+            dataframe = data.pop("dataframe")
+            if dataframe:
+                logger.warning(
```

Comment:
Hey @anakin87 we should be using the logger from Haystack right? So `from haystack import logging`

Reply:
Good catch. Done in https://github.com/deepset-ai/haystack-core-integrations/pull/1406/commits/8f7012d0e2f671997201ec7df59f490d7ad8ab2d

I've also noticed that we are not consistent across integrations. I'll create an issue for that.